__tests__/app.test.js

```diff
@@ -284,3 +284,37 @@ describe("/api/articles?topic=:topic", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id?comment_count", () => {
+  test("GET:200 Gets comment count for a certain article by id", () => {
+    return request(app)
+      .get("/api/articles/5?comment_count=true")
```

Comment:
We want comment_count _all_ the time now, not just on a query. Please go back and re-read the instructions. If earlier tests fail, amend them :) 

![Screenshot 2024-04-18 at 14 05 32](https://github.com/MuhammadSallu/my-news-project/assets/150431177/eaf14b67-8e5e-493b-9e8d-d9ddf60e6add)